lib/config/presets/internal/regex-managers.ts

```diff
@@ -4,14 +4,14 @@ import type { Preset } from '../types';
 
 export const presets: Record<string, Preset> = {
   dockerfileVersions: {
-    description: 'Update `_VERSION` variables in Dockerfiles.',
```

Comment:
why do you moved the description, please move back to original place

Reply:
Cz of the sort-keys comment above.